BSP/Simulator/Hardware/simulate.py

```diff
@@ -3,16 +3,20 @@
 from functools import partial
 
 import Switches
+import Contactor
 
 
 def update_buttons():
     """Periodically update the display state of buttons"""
     switches = Switches.read()
+    contactorState = Contactor.Read_Contactor()
```

Comment:
This function is for updating the buttons. You should make your own update function so that it's more modular